test/e2e_node/memory_manager_test.go

```diff
@@ -318,6 +332,14 @@ var _ = SIGDescribe("Memory Manager [Serial] [Feature:MemoryManager][NodeAlphaFe
 					memory:  "128Mi",
 				},
 			}
+
```

Comment:
can we create a separate test for it? something like
```golang
ginkgo.When("pod has init containers", func() {
    ginkgo.BeforeEach(//define init container)
    
    ginkgo.It("should succeed to start the pod", func() {....})
    
    ginkgo.JustAfterEach(// reset init containers, because otherwise all tests will run with init containers)
})

Reply:
sure, will be done